pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -560,3 +565,33 @@ func (p *ConfigurationEditFlags) AnyMutation(cmd *cobra.Command) bool {
 	}
 	return false
 }
+
+// Helper function for --scale
+func (p *ConfigurationEditFlags) scaleConversion(scale string) (scaleMin int, scaleMax int, err error) {
+	if len(scale) <= 2 {
```

Comment:
My preference would be to divide this into smaller private functions for each of the cases... might simply things. Also for each case returning early could also simply code since once case identified the result is there. No need for big if/else if/else.

Reply:
If I break up the helper function into multiple private functions it will complicate the cmd.Flags().Changed("scale") section, is that ok? I was trying to minimize how large that code block was going to be.

Reply:
Working on breaking this up next - I should be able to avoid complication in the cmd.Flags().Changed("scale") section.